### ISSUE TITLE
Add Number of implanted Fault A DTCs to user interface 

### ISSUE THREAD

**Issue by ericthomasswenson** [Equipment-and-Tool-Institute/j1939-84#1270]
Add a UI feature to the user input pop up to capture the number Fault A DTCs implanted for use in parts 3 and 4.

**Comment by ericthomasswenson** [comment]
Counts in parts 3 & 4 shall meet or exceed the user entered number as shown in C3 artifact.

**Comment by battjt** [comment]
Nov 14 2023, CARB assented to the concept of using the user interface to define the minimum number observed in fault A displays.

**Comment by karysg** [comment]
![faultACountScreenshot](https://github.com/user-attachments/assets/9bcaf883-729d-4336-a882-26f14bb1e861)

**Comment by ericthomasswenson** [comment]
Number of Implanted Fault A DTCs is preferred.

also group Fault A and Fault B content together in the following order:

Number of Implanted Fault A DTCs:      (known to all be two trip DTCs: Default to 1)

Number of Implanted Fault B DTCs:       (Default to 1)

Number of Fault B Dectection Trips:       (allowed to be 1 or Two)

**Comment by ericthomasswenson** [comment]
We confirm that individual failure criteria will also be updated to leverage these values.

Please don’t implement these piecemeal in test parts.

**Comment by ericthomasswenson** [comment]
Include failure criteria changes indicated by #1256 and #1257 as defined in the artifact with the rest of the multiple implant changes.

**Comment by anthonyhbest** [comment]
UI for Fault A DTC count is in place in version 3.1.30, but tool still fails for multiple DTCs seen.  (This may be expected behavior at this time)

**Comment by ericthomasswenson** [comment]
Correct, revised failure criteria for multiple faults is not yet implemented. UI got done early … there wasn’t a separate ticket for criteria … but the ticket comments reflect failure criteria implementation expectations. Comments on the UI “demo” were also added.Glad to here 3.1.30 fixed urgent content from 3.1.30Best Wishes,Eric SwensonSent from my iPadOn Sep 25, 2024, at 10:34 AM, anthonyhbest ***@***.***> wrote:﻿
UI for Fault A DTC count is in place in version 3.1.30, but tool still fails for multiple DTCs seen.  (This may be expected behavior at this time)
